### DISCUSSION TITLE
Error while running DAG using DBT and Airflow ==> dbt_snowflake://macros/adapters.sql

### DISCUSSION THREAD

**Discussion by sdy-volvo** [apache/airflow#26241] [Q&A]
### Apache Airflow Provider(s)

snowflake

### Versions of Apache Airflow Providers

I have copied my dbt project using the docker-compose.yaml for airflow and trying to run a DAG from locally docker deployed airflow and when i try to run the DAG it is giving the below error.

### Apache Airflow version

2.3.4

### Operating System

Win 10

### Deployment

Docker-Compose

### What happened

I have copied my dbt project using the docker-compose.yaml for airflow and trying to run a DAG from locally docker deployed airflow and when i try to run the DAG it is giving the below error.

![image](https://user-images.githubusercontent.com/88720402/189163334-84e3f4e9-b529-4024-84d4-36b012ae16e2.png)

### Are you willing to submit PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!
